src/Jira/Api.php

```diff
@@ -704,7 +704,7 @@ public function createRemotelink(
 	 * @param string       $method         Request method.
 	 * @param string       $url            URL.
 	 * @param array|string $data           Data.
-	 * @param boolean      $return_as_json Return results as JSON.
+	 * @param boolean      $return_as_array Return results as JSON.
```

Comment:
From GitHub UI looks like whole DocBlock needs to be reformatted (Ctrl+Alt+L in PhpStorm, when DocBlock is selected) to reflect new length of parameter name.